doc/api/child_process.md

```diff
@@ -1398,6 +1398,24 @@ then this will be `null`.
 `subprocess.stdout` is an alias for `subprocess.stdio[1]`. Both properties will
 refer to the same value.
 
+
+```js
+const { spawn } = require('child_process');
+
+const subprocess = spawn('ls');
+
+const ioStdout = subprocess.stdio[1];
+const stdoutStream = subprocess.stdout;
+
+ioStdout.on('data', (data) => {
+  console.log(`Received chunk ${data}`);
+});
+
+stdoutStream.on('data', (data) => {
```

Comment:
here, I would suggest just making this `subprocess.stdout` without the assignment to `stdoutStream`

Reply:
@jasnell Thank you! Corrected, also, remove some extra empty lines

Reply:
resolved in commit https://github.com/nodejs/node/pull/28402/commits/0be399a9001b4ab9968c4f9464ae41d15ed84b0b